ale_linters/php/php.vim

```diff
@@ -18,9 +19,9 @@ function! ale_linters#php#php#Handle(buffer, lines) abort
         " vcol is needed to indicate that the column is a character.
         call add(l:output, {
         \   'bufnr': a:buffer,
-        \   'lnum': l:match[2] + 0,
+        \   'lnum': l:match[3] + 0,
         \   'vcol': 0,
-        \   'col': 1,
+        \   'col': match(getline(l:match[3]), l:match[2]) + 1,
```

Comment:
I think i should have used `stridx()` here, which doesn't match against regular expressions.